projects/1.20/assets/vintage-delight/vintagedelight/lang/zh_cn.json

```
@@ -0,0 +1,186 @@
+    "item.vintagedelight.cheese_pasta": "奶酪意面",
+    "item.vintagedelight.cheese_slice": "奶酪切片",
+    "item.vintagedelight.organic_mash": "有机醪泥",
+    "item.vintagedelight.cheese_pizza_slice": "奶酪披萨切片",
+    "item.vintagedelight.meat_pizza_slice": "肉类披萨切片",
+    "item.vintagedelight.cucumber_salad": "黄瓜沙拉",
+    "item.vintagedelight.black_chefs_hat": "黑色厨师帽",
+    "item.vintagedelight.blue_chefs_hat": "蓝色厨师帽",
+    "item.vintagedelight.brown_chefs_hat": "棕色厨师帽",
+    "item.vintagedelight.cyan_chefs_hat": "青色厨师帽",
+    "item.vintagedelight.gray_chefs_hat": "灰色厨师帽",
+    "item.vintagedelight.green_chefs_hat": "绿色厨师帽",
+    "item.vintagedelight.light_blue_chefs_hat": "淡蓝色厨师帽",
+    "item.vintagedelight.light_gray_chefs_hat": "淡灰色厨师帽",
+    "item.vintagedelight.lime_chefs_hat": "黄绿色厨师帽",
+    "item.vintagedelight.magenta_chefs_hat": "品红色厨师帽",
+    "item.vintagedelight.orange_chefs_hat": "橘色厨师帽",
+    "item.vintagedelight.pink_chefs_hat": "粉红色厨师帽",
+    "item.vintagedelight.purple_chefs_hat": "紫色厨师帽",
+    "item.vintagedelight.red_chefs_hat": "红色厨师帽",
+    "item.vintagedelight.white_chefs_hat": "白色厨师帽",
+    "item.vintagedelight.yellow_chefs_hat": "黄色厨师帽",
+    "item.vintagedelight.salt_bucket": "盐桶",
+
+    "item.vintagedelight.salted_cod": "腌鳕鱼",
+    "item.vintagedelight.salted_salmon": "腌鲑鱼",
+    "item.vintagedelight.surstromming": "鲱鱼罐头",
+
+    "item.vintagedelight.pb_j": "花生酱和果酱三明治",
+    "item.vintagedelight.cheese_burger": "芝士汉堡",
+    "item.vintagedelight.deluxe_burger": "豪华汉堡",
+
+
+
+  "item.vintagedelight.mason_jar.tooltip": "可放置",
+  "farmersdelight.tooltip.nut_milk_bottle": "清除1个效果",
+  "gui.jei.category.fermenting.time.seconds": "腌制时间",
+  "vintagedelight.jei.info.salt_block": "放置在水源附近以产盐",
+  "vintagedelight.jei.info.evaporator": "放置在水源附近，以比盐块快得多的速度产盐",
+  "vintagedelight.jei.info.organic_compost": "腌制有机醪泥，用作强效肥料，或用于堆肥",
+  "vintagedelight.jei.info.cheese_wheel": "将 4个奶酪凝乳放入奶酪模具中以制作奶酪轮",
+
+
+
+  "vintagedelight.advancement.root": "腌渍乐事",
+  "vintagedelight.advancement.root.desc": "来腌制吧!",
+  "vintagedelight.advancement.harvest_salt": "咸咸的",
+  "vintagedelight.advancement.harvest_salt.desc": "收获盐",
+  "vintagedelight.advancement.place_fermenting_jar": "腌渍",
+  "vintagedelight.advancement.place_fermenting_jar.desc": "放置一个腌制罐",
+  "vintagedelight.advancement.place_salt_block": "产盐",
+  "vintagedelight.advancement.place_salt_block.desc": "放置在水源附近以产盐.",
+  "vintagedelight.advancement.place_evaporator": "蒸发器",
+  "vintagedelight.advancement.place_evaporator.desc": "放置一个被水包围的蒸发器，以比盐块快得多的速度产盐",
+  "vintagedelight.advancement.get_pickled_item": "在腌制中",
+  "vintagedelight.advancement.get_pickled_item.desc": "腌制一次蔬菜",
+  "vintagedelight.advancement.obtain_magic_peanut": "魔力花生!",
+  "vintagedelight.advancement.obtain_magic_peanut.desc": "也许是一只金鹅?",
+  "vintagedelight.advancement.obtain_century_egg": "获得世纪彩蛋",
+  "vintagedelight.advancement.obtain_century_egg.desc": "还没有为公布它做好准备",
+
+  "vintagedelight.advancement.get_jam_item": "粘糊糊'",
```

```suggestion
  "vintagedelight.advancement.get_jam_item": "黏糊糊",
```
原文`Jammin'`最后的单引号代表省音，是口语中把/ŋ/发成/n/的现象，不保留